src/components/inline-rendered-diff.jsx

```diff
@@ -16,11 +18,15 @@ import SandboxedHtml from './sandboxed-html';
  */
 export default class InlineRenderedDiff extends React.Component {
   render () {
-    const diff = this.props.diffData.combined || this.props.diffData.diff;
+    const diff = this.props.diffData.combined;
+    let transformDocument = (x) => x;
+    if (this.props.removeFormatting) {
+      transformDocument = removeStyleAndScript;
+    }
```

Comment:
I *think* you could just:

```js
const transformer = this.props.removeFormatting && removeStyleAndScript;
```

IIRC, `SandboxedHtml` doesn’t try and apply a transformer that’s `null` or `undefined`.